components/infra-proxy-service/server/orgs_test.go

```diff
@@ -1074,10 +1074,3 @@ func cleanupOrg(ctx context.Context, t *testing.T, cl infra_proxy.InfraProxyClie
 	_, err := cl.DeleteOrg(context.Background(), &deleteReq)
 	require.NoError(t, err)
 }
-
-func cleanupServer(ctx context.Context, t *testing.T, cl infra_proxy.InfraProxyClient, serverID string) {
```

Comment:
Move `cleanupServer` method to [servers_test.go](https://github.com/chef/automate/pull/3248/files#diff-26345778d259836272877d139e0983a4R514)